package.json

```diff
@@ -56,6 +53,7 @@
     "lint-staged": "13.2.2",
     "normalize-space-x": "4.1.2",
     "prettier": "2.8.8",
+    "ts-node": "10.9.1",
```

Comment:
Why ts-node is needed here?

Reply:
To support jest config with ts extension